README.md

```diff
@@ -28,11 +28,14 @@ This is a list of the Cargo features available for mbedtls-sys. Features in
 * *pthread* Enable mutex synchronization using pthreads.
 * *rdrand* Enable the RDRAND random number generator.
 * *spin_threading* Enable mutex synchronization using the spin crate.
+* *rust_threading* Enable mutex synchronization using the std::sync::Mutex.
+                   This feature is not suitable for no_std environment.
```

Comment:
“This feature is not suitable for no_std environment.” Remove this comment, obvious from use of `std` and `no_std`.